docs/user-guide/calendar/README.md

```diff
@@ -2,84 +2,124 @@
 
-You can filter the work packages in the calendar view by adding any number of filters, e.g. *Author*, *Due Date*, *Priority*, *Assignee*, etc.
+![Calendar filters](calendarFilters.png)
 
-Click the **Filter** button on the top right above the calendar to open the filter details.
+> **Note:** OpenProject also offers a module called Team planner that allows you to view modules in a calendar by assignee. [Learn more about Team planner](../team-planner "Documentation about Team planner").
 
-To add another filter criteria, select the drop down next to **+ Add filter** and choose a filter criteria.
+### Including other projects
 
-The number of different filter criteria is displayed on the filter button.
+Much like with work package tables, it is possible to display work packages from more other projects. Use the **Include projects** dialog, where you can select/unselect from a list of projects and sub-projects. To automatically include all sub-projects for each project you chose to select, check the **Include all sub-projects** box at the bottom of the dialog.
 
-![filter-calendar](filter-calendar.png)
+### Embedding a calendar in the project overview
 
 You can also display the calendar on the dashboard in the [project overview](../project-overview/#calendar-widget).
 
-## Zen mode
+### Zen mode
+
+You can display the calendar full screen mode by hiding the navigation bar  and sidebar to work free of distractions. Click the **Activate zen mode** button on the top of the calendar. Press the escape key (**Esc**) to exist Zen mode.
+
+## Subscribing to a calendar
+
+OpenProject allows you to subscribe to and access any of your calendars using an external client that supports the .ics format.
 
-You can display the calendar in a full screen view to eliminate header and project menu and work distraction free.
+This makes it possible for you to keep an eye on your project schedule from any compatible device (including mobile devices) without having to connect to your OpenProject instance and always remain up to date with relevant work packages.
 
-Click the **Activate zen mode** button on the top of the calendar.
+> **Note:** You can only subscribe to saved calendars. If you are creating a new calendar, please save it before you are able to subscribe to it.
 
-Click Escape (**Esc**) to leave the Zen mode.
+![Subscribe to calendar in the More menu](subscribeToCalendar.png)
 
-![zen-calendar](zen-calendar.png)
+To subscribe to a calendar:
 
-## Project overarching calendar
+1.  Click on the **[⋮]** (more) on the toolbar and select **Subscribe to calendar**.
+2.  In the modal that appears, give this calendar a name. We recommend naming it based on where you will be subscribing to this calendar from ("personal phone" or "work tablet" for example).
```

Comment:
@psatyal probably we need to specify that the name used can be only used once.

Reply:
Good point. Updated!